ledger-service/http-json-cli/src/main/scala/com/daml/http/Config.scala

```diff
@@ -46,6 +45,7 @@ private[http] final case class Config(
     metricsReporter: Option[MetricsReporter] = None,
     metricsReportingInterval: FiniteDuration = MetricsConfig.DefaultMetricsReportingInterval,
     surrogateTpIdCacheMaxEntries: Option[Long] = None,
+    histograms: Seq[HistogramDefinition] = Seq.empty,
```

Comment:
Where do we have the config options documented?

Reply:
We don't, this is an internal option only, that ca be used to adjust on the flight before we make more permanent changes.